src/components/about-us/opening/background.js

```diff
@@ -0,0 +1,179 @@
+import { screen } from '../utils/screen'
+import { intro } from '../constants/data/introduction'
+import billboard from '../../../../static/asset/about-us/billboard.png'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const TriangleSizes = {
```

Comment:
use camelCase.
variable starting with uppercase letter is for class declaration.